docs/PROVISIONING_AWS.md

```diff
@@ -0,0 +1,102 @@
+# OpenShift on AWS EC2 using CASL
+
+## Local Setup (one time, only)
+
+> **NOTE:** These steps are a canned set of steps serving as an example, and may be different in your environment.
+
+In addition to _cloning this repo_, you'll need the following:
+
+* Access to an AWS account with the proper policies to create resources (see details below)
+* Docker installed
+  * RHEL/CentOS: `yum install -y docker`
+  * Fedora: `dnf install -y docker`
+  * **NOTE:** If you plan to run docker as yourself (non-root), your username must be added to the `docker` user group.
+
+```
+cd ~/src/
+git clone https://github.com/redhat-cop/casl-ansible.git
+```
+
+* Download/untar `openshift-ansible` for use as part of the install. Make sure to do so within the same directory as above (i.e.: `~/src`), and either rename the directory to `openshift-ansible` (or create symlink to it - see example below). See our [Compatibility Matrix](./README.md#compatability-matrix) for versions / urls to be used for the download.
```

Comment:
'Compatibility Matrix' link is broken. The relative path must be changed

Reply:
Sure, will fix it, but at soon as we move to 3.7, this entire matrix will be removed as we'll use `ansible-galaxy` instead, so not a huge priority as-is